src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt

```diff
@@ -0,0 +1,738 @@
+                "\n    §9> §rIgnored Blocks:"
+        for (b in ignoreBlocks) {
+            message += "\n        §9> §7${b!!.localizedName}"
+        }
+        MessageSendHelper.sendChatMessage(message)
+    }
+
+    private fun printEnable() {
+        if (stats.value) {
+            if (buildDirectionSaved == 1 || buildDirectionSaved == 3 || buildDirectionSaved == 5 || buildDirectionSaved == 5 || buildDirectionSaved == 7) {
+                MessageSendHelper.sendChatMessage("$chatName Module started." +
+                        "\n    §9> §7Direction: §a" + directions[buildDirectionSaved] + "§r" +
+                        "\n    §9> §7Coordinates: §a" + mc.player.positionVector.x.roundToInt() + ", " + mc.player.positionVector.z.roundToInt() + "§r" +
+                        "\n    §9> §7Baritone mode: §a" + baritoneMode.value + "§r")
+            } else {
+                MessageSendHelper.sendChatMessage("$chatName Module started." +
+                        "\n    §9> §7Direction: §a" + directions[buildDirectionSaved] + "§r" +
+                        "\n    §9> §7Coordinate: §a" + buildDirectionCoordinate + "§r" +
+                        "\n    §9> §7Baritone mode: §a" + baritoneMode.value + "§r")
+            }
+        } else {
+            MessageSendHelper.sendChatMessage("$chatName Module started.")
+        }
+    }
+
+    private fun printDisable() {
+        if (stats.value) {
+            MessageSendHelper.sendChatMessage("$chatName Module stopped." +
+                    "\n    §9> §7Placed blocks: §a" + totalBlocksPlaced + "§r" +
+                    "\n    §9> §7Destroyed blocks: §a" + totalBlocksDestroyed + "§r" +
+                    "\n    §9> §7Distance: §a" + totalBlocksDistance + "§r")
+        } else {
+            MessageSendHelper.sendChatMessage("$chatName Module stopped.")
+        }
+    }
+
+    fun getNextBlock(): BlockPos {
+        return when (buildDirectionSaved) {
+            0 -> { currentBlockPos.north() }
+            1 -> { currentBlockPos.north().east() }
+            2 -> { currentBlockPos.east() }
+            3 -> { currentBlockPos.south().east() }
+            4 -> { currentBlockPos.south() }
+            5 -> { currentBlockPos.south().west() }
+            6 -> { currentBlockPos.west() }
+            else -> { currentBlockPos.north().west() }
+        }
+    }
+
+    private fun lookInWalkDirection() {
+        // set head rotation to get max walking speed
+        when (buildDirectionSaved) {
+            0 -> { mc.player.rotationYaw = -180F }
+            1 -> { mc.player.rotationYaw = -135F }
+            2 -> { mc.player.rotationYaw = -90F }
+            3 -> { mc.player.rotationYaw = -45F }
+            4 -> { mc.player.rotationYaw = 0F }
+            5 -> { mc.player.rotationYaw = 45F }
+            6 -> { mc.player.rotationYaw = 90F }
+            else -> { mc.player.rotationYaw = 135F }
+        }
+        mc.player.rotationPitch = 0F
+    }
```

Comment:
see my above comment about the util, also this shouldn't be using arbitrary Ints